src/tools/rbd/action/Import.cc

```diff
@@ -526,7 +526,7 @@ int execute_diff(const po::variables_map &vm,
 }
 
 Shell::Action action_diff(
-  {"import-diff"}, {}, "Import an incremental diff.", "", &get_arguments_diff,
+  {"import-diff"}, {}, "Apply an incremental diff to image HEAD, then create a snapshot.", "", &get_arguments_diff,
```

Comment:
This line is too long now, need to wrap:
```suggestion
  {"import-diff"}, {},
  "Apply an incremental diff to image HEAD, then create a snapshot.", "",
  &get_arguments_diff, &execute_diff);
```